src/ui/buttons/tagline.jsx

```diff
@@ -30,7 +30,7 @@ export function TagLine({ fundingSource, locale, multiple, nonce, personalizatio
     return (
         <div class={ CLASS.TAGLINE }>
             {
-                (tagline && !multiple)
```

Comment:
Just do double check have you ensured that the default multiple buttons tagline still works after this?

Reply:
Yes, it's working! `Two easy ways to pay tagline` is being shown, which is the default tagline